src/boot.tsx

```diff
@@ -22,6 +22,22 @@ export default (element: HTMLDivElement, hub: Hub) => {
     applyMiddleware(createEpicMiddleware(rootEpic))
   );
 
+  const getPackages = (packages: string) => {
+    if (packages){
+      return "\nfrom dcl_package_manager import install_packages, get_registry; install_packages(["+formatPackages(packages)+"], get_registry('/registry/registry.json'))";
+    }
+    return "";
+  };
+
+  const formatPackages = (packages: string) => {
+    let r = packages.split(",");
+    let output = "";
+    for (let i=0; i<r.length; i++){
+      output += '"'+r[i]+'", ';
+    } 
+    return output;
```

Comment:
You can combine split with `reduce` (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce) here and make this function almost a oneliner.

Reply:
Nice one, thanks, didn't know about reduce